Projekty/Projekt1/Degorski_Kaminski/pro1.ipynb

```diff
@@ -0,0 +1,1616 @@
+{
```

Comment:
<p>Jak najbardziej, ale dobrze że to sprawdziliście. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>